scripts/test.sh

```diff
@@ -2,12 +2,79 @@
 set -euo pipefail
 IFS=$'\n\t'
 set -x
```

Comment:
You should enclose the actual useful commands inside the `set -x` `set +x` statements. It avoids printing things like this:

```
+ COVERAGE=--no-cov
+ EXEC_COMMAND=true
+ CHECK_MIGRATIONS=true
+ N_CPU=auto
+ PYTEST_ARGS=("")
+ [[ 0 -gt 0 ]]
+ '[' true = true ']'
+ echo 'Checking for missing migrations...'
Checking for missing migrations...
```
The last line should print, but the ones before aren't useful.

My original goal was to print out the actual commands being run so that someone can also copy, modify and run them. Or just use it to learn the tool's real syntax.